app/controllers/concerns/sufia/collections_controller_behavior.rb

```diff
@@ -30,6 +30,16 @@ def show
       presenter
     end
 
+    def create
+      super
+      ActiveFedora::Base.find(params[:batch_document_ids]).each(&:update_index) unless params[:batch_document_ids].nil?
+    end
+
+    def update
+      super
+      ActiveFedora::Base.find(params[:batch_document_ids]).each(&:update_index) unless params[:batch_document_ids].nil?
```

Comment:
Since this method is identical in both `#create` and `#update`, could you extract it to a private method and call it from both places?